homeassistant/components/lcn/manifest.json

```diff
@@ -9,5 +9,5 @@
   "iot_class": "local_polling",
   "loggers": ["pypck"],
   "quality_scale": "bronze",
-  "requirements": ["pypck==0.9.3", "lcn-frontend==0.2.7"]
+  "requirements": ["pypck==0.9.4", "lcn-frontend==0.2.7"]
```

Comment:
mypkg isn't the name of the library package:

https://github.com/alengwenus/pypck/commit/fd06c949da35f8bce59286182e79f34485725a40

It should be `pypck`.

Reply:
This could be the reason for mypy failing here:
https://github.com/home-assistant/core/actions/runs/19479921206/job/55749409624?pr=156831